packages/core/src/openapi/openapi-request-builder.ts

```diff
@@ -45,11 +47,11 @@ export class OpenApiRequestBuilder<ApiT, FnT extends keyof ApiT> {
 
   /**
    * Execute request.
-   * @param destination  Destination to execute the request against.
+   * @param destination Destination to execute the request against.
```

Comment:
perhaps you can adjust the text form the `execute` and `executeRaw` a bit to make clear what the differences are. and when to use it.

I know it is in the return parameters but it is always nice to have a bit text in the beginning.